tests/components/brunt/test_config_flow.py

```diff
@@ -0,0 +1,157 @@
+"""Test the Brunt config flow."""
+from unittest.mock import patch
+
+import pytest
+
+from homeassistant import config_entries, data_entry_flow
+from homeassistant.components.brunt.const import DOMAIN
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+
+from tests.common import MockConfigEntry
+
+CONF = {CONF_USERNAME: "test-username", CONF_PASSWORD: "test-password"}
+
+
+async def _flow_submit(hass, context={"source": config_entries.SOURCE_USER}, data=CONF):
+    """Submit a flow to hass."""
+    return await hass.config_entries.flow.async_init(
+        DOMAIN,
+        context=context,
+        data=data,
+    )
+
+
+async def test_form(hass):
+    """Test we get the form."""
+    result = await _flow_submit(hass, data=None)
+    assert result["type"] == "form"
+    assert result["errors"] is None
+
+    with patch(
+        "homeassistant.components.brunt.config_flow.validate_input", return_value=None
```

Comment:
Please don't patch our code in the config flow. Patch the library. We want all code in the config flow to be tested.